.changelog/10391.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+proxycfg: Explicit upstreams in other datacenters shouldn't be deleted when transparent proxy is enabled, and when they're not in the seen services list of explicit and implicit upstreams within a datacenter.
```

Comment:
not blocking, but I wonder if this might be too much detail for a user reading the changelog. They might not have enough context to understand what this means.

How did this bug manifest to users?

I'm not sure I understand this change well enough to suggest an improvement, but maybe something roughly like: "watch upstreams from remote datacenters to ..."

My assumption is that the "do not delete" means that we continue to watch them for changes?

Reply:
That assumption is correct.

Will update the message:
```suggestion
proxycfg: Ensure that endpoints for explicit upstreams in other datacenters are watched in transparent mode.
```

Reply:
Thank you that looks good!